src/styles/blocks/product-card.scss

```diff
@@ -0,0 +1,104 @@
+.product-card {
+  display: flex;
+  width: 198px;
+  height: 406px;
```

Comment:
you dont need to hardcode height, it must stretch from content

Reply:
https://github.com/mate-academy/layout_catalog/pull/3507/files#r1282206569

didn't fix it